pages/support/[category]/index.js

```diff
@@ -1,37 +1,16 @@
 import { useState, useEffect, useContext } from 'react'
 import styled from 'styled-components'
+import clsx from 'clsx'
 import * as R from 'ramda'
 import useModal from 'use-react-modal'
-import { LIKES, NAME, RECENT } from '../../../lib/constants'
+import { SUPPORT_FILTER, LOCATION_FILTER } from '../../../lib/constants'
```

Comment:
heroku build error says this isn't being exported from constants.js

Reply:
Fixed!